code/game/objects/items/devices/radio/radio_objects.dm

```diff
@@ -131,6 +131,14 @@ GLOBAL_LIST_EMPTY(deadsay_radio_systems)
 		return
 	ui_interact(user)
 
+/obj/item/radio/AltClick(mob/user)
+	if(user.stat || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user) || !istype(user))
+		return
+
+	ToggleBroadcast()
+	to_chat(user, "<span class='notice'>You <b>[isBroadcasting() ? "enable" : "disable"]</b> [src]'s hotmic!</span>")
```

Comment:
Why even use a proc for this? You could replace it with the var `broadcasting`

Reply:
Im a silly goose.